Geometry/VeryForwardGeometryBuilder/src/DetGeomDesc.cc

```diff
@@ -27,8 +28,14 @@ DetGeomDesc::DetGeomDesc( DDFilteredView* fv ) :
   m_name((( fv->logicalPart()).ddname()).name()),
   m_params((( fv->logicalPart()).solid()).parameters()),
   m_copy( fv->copyno()),
-  m_z( fv->geoHistory().back().absTranslation().z())
-{}
+  m_z( fv->geoHistory().back().absTranslation().z()),
```

Comment:
@jan-kaspar - please, move access to **DDFilteredView** out of the constructor and provide all these parameters in an explicit **DetGeomDesc** constructor.

Reply:
@ianna , this is the back-port to legacy Run-2 release. Your request is fully valid to the current master software but not for back-porting of already merged PRs.

Reply:
Several updates (in master) to PPS geometry are [planned](https://docs.google.com/spreadsheets/d/1Yv_2uDdqvFGLTjspm60fnUMQfLr3geuNF_xXfpd_uWI/edit?usp=sharing) with ETA by March.

Reply:
> Several updates (in master) to PPS geometry are [planned](https://docs.google.com/spreadsheets/d/1Yv_2uDdqvFGLTjspm60fnUMQfLr3geuNF_xXfpd_uWI/edit?usp=sharing) with ETA by March.

@cvuosalo - FYI